cli/cmd/root.go

```diff
@@ -76,10 +77,19 @@ func runCmd(ctx context.Context, ver config.Version) error {
 		Version: ver,
 	}
 
+	format := printer.Human
+	p := printer.NewPrinter(&format)
+
+	// Create cmdutil Helper
+	ch := &cmdutil.Helper{
+		Config:  cfg,
+		Printer: p,
+	}
+
 	// Check version
 	err := update.CheckVersion(ctx, cfg.Version.Number)
 	if err != nil {
-		fmt.Printf("Warning: version check failed: %v\n", err)
+		p.PrintlnWarn(fmt.Sprintf("Warning: version check failed: %v\n", err))
```

Comment:
Extra `Sprintf`.

Reply:
Keeping as is for now as `PrintlnWarn` requires string as args